firewood/src/merkle/node.rs

```diff
@@ -196,15 +196,23 @@ impl From<NodeType> for Node {
     }
 }
 
+bitflags! {
+    struct NodeAttributes: u8 {
+        const ROOT_HASH_VALID      = 0b001;
+        const IS_ENCODED_BIG_VALID = 0b010;
+        const LONG                 = 0b100;
+    }
+}
+
 impl Node {
-    const BRANCH_NODE: u8 = 0x0;
-    const EXT_NODE: u8 = 0x1;
-    const LEAF_NODE: u8 = 0x2;
+    const BRANCH_NODE: u8 = 0;
+    const LEAF_NODE: u8 = 1;
+    const EXT_NODE: u8 = 2;
```

Comment:
Order here changes on purpose. The important part is that it is the same as below